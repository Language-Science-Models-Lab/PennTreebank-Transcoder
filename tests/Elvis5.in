
(S @(NP @(NNP @Elvis)) (VP @(VBZ @is) (ADJP-PRD~ @(JJ @dead))) (. @.))

# tests phrase rules 1 and 13
# tests clause rule 5


