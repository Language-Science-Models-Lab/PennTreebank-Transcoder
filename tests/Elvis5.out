
[Elvis [is dead]]:type(type(Elvis, _)@type(type(is, _)@type(dead, _), _), s)
